Google.Api.Gax.Grpc/GrpcNetClientAdapter.cs

```diff
@@ -16,22 +17,40 @@ namespace Google.Api.Gax.Grpc
     /// </summary>
     public sealed class GrpcNetClientAdapter : GrpcAdapter
     {
+        private Action<global::Grpc.Net.Client.GrpcChannelOptions> _optionsConfigurationAction;
+
         // Note: this is "Default" rather than "Instance" as we expect to have other factory methods later, e.g. accepting
         // an HTTP client factory.
+        // TODO: Rename now that there's GrpcAdapter.DefaultAdapter?
```

Comment:
I think it's fine to have `GrpcAdapter.DefaultAdapter` and then this one to be `Fallback` and for GrpcCore and Rest, `Instance`.